File: apps/problems/max-element/boilerplate-full/function.js

  /**
   * @param {number} size
 * @param {Array} arr
   * @return {number}
   */
  var maxElement = function(size, arr) {
      // Write your code here
  };
  
  const input = require('fs').readFileSync('/dev/stdin').toString().trim().split('\n');
  const size = Number(input.shift());
const arr = input.shift().split(',').map(Number);
  const result = maxElement(size, arr);
  console.log(JSON.stringify(result));
      

File: apps/problems/max-element/boilerplate-full/function.java

  import java.util.*;
  import java.util.stream.*;
  
  class Solution {
      public int maxElement(int size, int[] arr) {
          // Write your code here
          return null; // Replace with actual result
      }
  
      public static void main(String[] args) {
          Scanner scanner = new Scanner(System.in);
          int size = Integer(scanner.nextLine());
        int[] arr = Arrays.stream(scanner.nextLine().split(",")).mapToint(Integer::valueOf).toArray();
          Solution solution = new Solution();
          int result = solution.maxElement(size, arr);
          System.out.println(result);
          scanner.close();
      }
  }
      